### PR TITLE
Poc4 version2

### PR DESCRIPTION
**Pull request by EdytaSki**
<h1> Pull Request Template </h1>

<h2> Description </h2>

Please include a short resume of the changes and what is the purpose of PR. Any relevant information should be added to help:
* **QA Team** (Quality Assurance) with tests.
* **reviewers** to understand what are the stakes of the pull request.

Fixes # (issue)

<h2> Type of change </h2>

- [ ] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

<h2> Target serie </h2>

- [ ] 2.8.x
- [ ] 18.10.x
- [x] 19.04.x (master)

<h2> How this pull request can be tested ? </h2>

Please describe the **procedure** to verify that the goal of the PR is matched. Provide clear instructions so that it can be **correctly tested**.

Any **relevant details** of the configuration to perform the test should be added.

<h2> Checklist </h2>

<h5> Community contributors & Centreon team </h5>

- [x] I followed the **coding style guidelines** provided by Centreon
- [x] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [x] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have made corresponding changes to the **documentation**.
- [ ] I have **rebased** my development branch on the base branch (master, maintenance).

<h5> Centreon team only </h5>

- [ ] I have made sure that the **unit tests** related to the story are successful.
- [ ] I have made sure that **unit tests covers 80%** of the code written for the story.
- [x] I have made sure that **acceptance tests** related to the story are successful (**local and CI**)

### CODE REVIEW

File: www/front_src/src/components/navigation/index.js [hunk 1]
```diff
@@ -268,11 +241,17 @@ class NavigationComponent extends Component {
                                     if (levelFourProps.label) {
                                       return (
                                         <li
-                                          class={"collapsed-level-item" + (this.isActive(pageId, levelFourUrl) ? " active" : "")}
+                                          onClick={() => {
+                                            this.state.active ?
+                                              this.activeCurrentLevel(levelOneKey, levelTwoKey)
```
Comment (kduret) [inline www/front_src/src/components/navigation/index.js]: bad coding style. Should be : 
expression
  ? result1
  : result2

File: www/front_src/src/components/navigation/index.js [hunk 2]
```diff
@@ -197,69 +174,65 @@ class NavigationComponent extends Component {
...
-                          class={
-                            "collapsed-item" +
-                            (levelTwoProps.collapsed || (this.isActive(pageId, levelTwoUrl))
-                              ? " active"
-                              : ""
-                            )
+                          className={
+                            `collapsed-item
+                            ${levelTwoProps.collapsed || (this.isActive(pageId, levelTwoUrl))
+                            ?
```
Comment (kduret) [inline www/front_src/src/components/navigation/index.js]: bad coding style. Should be : 
expression
  ? result1
  : result2

Comment (EdytaSki) [reply www/front_src/src/components/navigation/index.js]: done

File: www/front_src/src/components/navigation/index.js (hunk 1 above)
Comment (EdytaSki) [reply www/front_src/src/components/navigation/index.js]: done

Comment (danielhalupka) [reply www/front_src/src/components/navigation/index.js]: Did not see this, indeed. But there is no need for the ternary and active should be deconstructed.

**Pull request merged**